packages/playground/src/app.jsx

```diff
@@ -602,7 +602,7 @@ class Playground extends Component {
               react-jsonschema-form
             </a>
             .
-            {process.env.SHOW_NETLIFY_BADGE === "true" && (
+            {import.meta.env.SHOW_NETLIFY_BADGE === "true" && (
```

Comment:
I know very little about `vite` but I'm guessing that they have a different way of getting environment variables?